cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/ExamplePowerVmAllocationPolicyMigrationStaticThreshold.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.cloudsimplus.migration;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import org.cloudbus.cloudsim.allocationpolicies.power.PowerVmAllocationPolicyMigrationAbstract;
+import org.cloudbus.cloudsim.hosts.power.PowerHost;
+import org.cloudbus.cloudsim.selectionpolicies.power.PowerVmSelectionPolicy;
+import org.cloudsimplus.sla.readJsonFile.SlaMetric;
+import org.cloudsimplus.sla.readJsonFile.SlaReader;
+
+/**
+ *
+ * @author raysaoliveira
+ */
+public class ExamplePowerVmAllocationPolicyMigrationStaticThreshold extends PowerVmAllocationPolicyMigrationAbstract {
```

Comment:
This class doesn't have to exist. You are duplicating an entire class that doesn't have to be even changed. The existing PowerVmAllocationPolicyMigrationStaticThreshold already works as it must work. What you have to do is just to instantiate such a class in your example and inside the example is where you have to load the thresholds from the SLA contract.

Further we can find an appropriate place to load these parameters, but it doesn't have to be inside the allocation policy.